pkg/pool-manager/pool.go

```diff
@@ -31,6 +31,7 @@ const (
 	RuntimeObjectFinalizerName = "k8s.v1.cni.cncf.io/kubeMacPool"
 	networksAnnotation         = "k8s.v1.cni.cncf.io/networks"
 	networksStatusAnnotation   = "k8s.v1.cni.cncf.io/networks-status"
+	vmWaitConfigMapName        = "kubemacpool-vm-configmap"
```

Comment:
what is this CM good for?

Reply:
This CM saved the allocated mac addresses in waiting state.

We need to use it in case of a race condition that our manager go down and It have that types of allocations so the new leader can continue from the same state.

Reply:
Sounds dangerous synchronisation-wise, but that's not different from the rest of kubemacpool... :D